css/components/toast.css

```diff
@@ -0,0 +1,76 @@
+.toast-container {
+  position: fixed;
+  top: 1rem;
+  right: 0;
+  z-index: 4;
+}
+
+.toast-container.hide-display {
+  display: none;
+}
+
+.toast-icon {
+  width: 2.4rem;
+  height: 2.4rem;
+}
+
+.toast {
+  border-radius: 0.5rem;
+  max-width: 60rem;
+  width: 100%;
+  font-size: 1.8rem;
+  display: flex;
+  align-items: center;
+  justify-content: start;
```

Comment:
Default value of justify-content is flex-start so you can remove it

Reply:
fixed